app/src/main/res/xml/fragment_neuro_settings.xml

```diff
@@ -4,7 +4,7 @@
 
     <EditTextPreference
         android:defaultValue="3"
-        android:digits="0123456789"
```

Comment:
This will prevent user from entering '0' from the input. What if user wants to enter '10'?